src/source/PeerConnection/SessionDescription.c

```diff
@@ -286,6 +301,9 @@ STATUS setTransceiverPayloadTypes(PHashTable codecTable, PHashTable rtxTable, PD
             (pKvsRtpTransceiver->transceiver.direction == RTC_RTP_TRANSCEIVER_DIRECTION_SENDRECV ||
              pKvsRtpTransceiver->transceiver.direction == RTC_RTP_TRANSCEIVER_DIRECTION_SENDONLY)) {
             CHK_STATUS(hashTableGet(codecTable, pKvsRtpTransceiver->sender.track.codec, &data));
+            if (pKvsRtpTransceiver->sender.track.codec == RTC_RTX_CODEC_H265) {
+                data = DEFAULT_PAYLOAD_H265;
+            }
```

Comment:
This seems asymmetric compared to what we do for other codecs.  Why do we need to do this?

Reply:
Removed. We don't.